team_unified_prs.rb

```diff
@@ -62,5 +62,6 @@ def positional_arg_or_env_split(idx, env_var)
     add_prs.call(Github.open_pull_requests_for_repo(repo))
   end
 
-  Github.puts_multiple_pull_requests(all_prs.values, { indexed: true })
+  prs_reverse_date = all_prs.values.sort{|a,b| b["createdAt"] <=> a["createdAt"]}
```

Comment:
`sort_by` is generally preferred for this type of thing, since will take fewer iterations

```suggestion
  prs_reverse_date = all_prs.values.sort_by { |pr| pr["createdAt"] }
```